src/neo/Wallets/NEP6/NEP6Account.cs

```diff
@@ -83,5 +85,49 @@ public bool VerifyPassword(string password)
                 return false;
             }
         }
+
+        /// <summary>
+        /// Cache draft nep2key during wallet password changing process. Should not be called alone for a single account
+        /// </summary>
+        internal bool ChangePasswordPrepare(string password_old, string password_new)
+        {
+            if (WatchOnly) return true;
+            KeyPair keyTemplate = null;
+            if (nep2key == null)
+            {
+                if (key == null)
+                {
+                    return true;
+                }
+            }
+            else
+            {
+                try
+                {
+                    keyTemplate = new KeyPair(Wallet.GetPrivateKeyFromNEP2(nep2key, password_old, wallet.Scrypt.N, wallet.Scrypt.R, wallet.Scrypt.P));
+                }
+                catch
+                {
+                    return false;
+                }
+            }
+            nep2KeyNew = keyTemplate.Export(password_new, wallet.Scrypt.N, wallet.Scrypt.R, wallet.Scrypt.P);
```

Comment:
`keyTemplate` could be null in line 102 (with `nep2key==null && key!=null` )

Reply:
Thanks for reminding, have updated:)